services/core-api/app/config.py

```diff
@@ -103,15 +103,18 @@ def JWT_ROLE_CALLBACK(jwt_dict):
     NROS_CLIENT_ID = os.environ.get('NROS_CLIENT_ID', None)
     NROS_TOKEN_URL = os.environ.get('NROS_TOKEN_URL', None)
 
-    # VFCBC
+    # vFCBC
     VFCBC_CLIENT_SECRET = os.environ.get('VFCBC_CLIENT_SECRET', None)
     VFCBC_CLIENT_ID = os.environ.get('VFCBC_CLIENT_ID', None)
 
-    # NRIS Remote API
+    # NRIS
     NRIS_REMOTE_CLIENT_SECRET = os.environ.get('NRIS_REMOTE_CLIENT_SECRET', None)
     NRIS_REMOTE_CLIENT_ID = os.environ.get('NRIS_REMOTE_CLIENT_ID', None)
     NRIS_REMOTE_TOKEN_URL = os.environ.get('NRIS_REMOTE_TOKEN_URL', None)
 
+    # OrgBook
+    ORGBOOK_API_URL = 'https://orgbook.gov.bc.ca/api/v2/'
```

Comment:
Can we have this as an environment variable as well? Good practice, plus once we become an issuer we might be running a local orgbook instance for development.

Reply:
Sure, although, when we are running our own local instance I think we will still need this URL in order to retrieve data stored in VON/OrgBook.